vernac/declareObl.ml

```diff
@@ -532,7 +532,7 @@ let obligation_terminator entries uctx { name; num; auto } =
     (* Ensure universes are substituted properly in body and type *)
     let body = EConstr.to_constr sigma (EConstr.of_constr body) in
     let ty = Option.map (fun x -> EConstr.to_constr sigma (EConstr.of_constr x)) ty in
-    let ctx = Evd.evar_universe_context sigma in
+    let uctx = Evd.evar_universe_context sigma in
```

Comment:
Could just remove that line, we haven't modified sigma

Reply:
Indeed, fixed thanks.